docs/aws/audit/iamcompliance/rules/root_account_access_keys.mdx

```diff
@@ -18,6 +18,29 @@ Security
 
 CBP, PCIDSS, SOC2, CIS, ISO27001, HIPAA
 
+### Remediation
+
+How to remove any active access keys created for your AWS root account?
+
+#### Using AWS Console
+
+1. Log in to the AWS Management Console using your root account credentials.
+2. Navigate to the IAM service by searching for "IAM" in the AWS services search bar and selecting "IAM" from the results.
+3. In the IAM dashboard, click on "Users" in the left sidebar.
+4. Locate the root account user and click on its username.
```

Comment:
wherever it's self explanatory like in this case (there can be only one root account :) ), let's not add the additional line.